### PR TITLE
spellcheck

### PR DESCRIPTION
**Pull request by ShizCalev**

### CODE REVIEW

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -7069,7 +7069,7 @@
 /area/maintenance/fore/secondary)
 "apv" = (
 /obj/machinery/door/airlock/maintenance{
-	name = "Fitness Maitenance";
```
Comment (Tacolizard) [inline _maps/map_files/BoxStation/BoxStation.dmm]: holy fuck

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 2]
```diff
@@ -11151,7 +11151,7 @@
 /area/crew_quarters/fitness)
 "azi" = (
 /obj/machinery/door/airlock/maintenance{
-	name = "Garden Maintenace";
```
Comment (Tacolizard) [inline _maps/map_files/BoxStation/BoxStation.dmm]: my sides

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -57835,7 +57835,7 @@
 /obj/machinery/conveyor_switch/oneway{
 	convdir = -1;
 	id = "garbage";
-	name = "disposal coveyor"
```
Comment (Tacolizard) [inline _maps/map_files/BoxStation/BoxStation.dmm]: are in orbit

File: _maps/map_files/Cerestation/cerestation.dmm [hunk 4]
```diff
@@ -27365,7 +27365,7 @@
 "bae" = (
 /obj/machinery/power/apc{
 	dir = 1;
-	name = "Port Asteroid Maintence APC";
```
Comment (Tacolizard) [inline _maps/map_files/Cerestation/cerestation.dmm]: ok this one is just sad

**Comment by banthebantz** [comment]
LERN 2 SPEL

**Comment by ShizCalev** [comment]
fuk.

sec.

**Pull request closed without merging**